src/redux/contact/contact.sagas.js

```diff
@@ -0,0 +1,149 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+import { push } from 'connected-react-router';
+import {
+  getContacts,
+  deleteContact,
+  getContactById,
+  addContact,
+  updateContact
+} from './contact.operations';
+
+import {
+  setContacts,
+  setContactsLoading,
+  setContact,
+  setContactsError,
+  setContactsCurrentPage,
+  setContactsPagesCount
+} from './contact.actions';
+
+import {
+  GET_CONTACTS,
+  DELETE_CONTACT,
+  ADD_CONTACT,
+  UPDATE_CONTACT,
+  GET_CONTACT
+} from './contact.types';
+
+import {
+  setSnackBarSeverity,
+  setSnackBarStatus,
+  setSnackBarMessage
+} from '../snackbar/snackbar.actions';
+
+import { config } from '../../configs';
+
+const {
+  SUCCESS_ADD_STATUS,
+  SUCCESS_DELETE_STATUS,
+  SUCCESS_UPDATE_STATUS
+} = config.statuses;
+
+const { skip, limit, contactsPerPage } = config.paginationPayload;
+
+function* handleContactsLoad({
+  payload = {
+    skip: 1,
+    limit: 1,
+    contactsPerPage: 1
+  }
+}) {
+  try {
+    yield put(setContactsLoading(true));
+    const contacts = yield call(getContacts, payload.skip, payload.limit);
+    yield put(
+      setContactsPagesCount(Math.ceil(contacts.count / payload.contactsPerPage))
+    );
+    yield put(setContacts(contacts.items));
+    yield put(setContactsLoading(false));
+  } catch (error) {
+    yield call(handleContactsError, error);
+  }
+}
+
+function* handleContactLoad({ payload }) {
+  try {
+    yield put(setContactsLoading(true));
+    const contact = yield call(getContactById, payload);
+    yield put(setContact(contact));
+    yield put(setContactsLoading(false));
+  } catch (error) {
+    yield call(handleContactsError, error);
+  }
+}
+
+function* handleAddContact({ payload }) {
+  try {
+    yield put(setContactsLoading(true));
+    yield call(addContact, payload);
+    const contacts = yield call(getContacts, skip, limit);
```

Comment:
Do you really need to fetch all contacts again after creation? I think it is much easier to just add a newly created one to the redux instead of making another request to the backend